poppy/zernike.py

```diff
@@ -1170,8 +1177,8 @@ def opd_from_zernikes(coeffs, basis=zernike_basis_faster, aperture=None, outside
     return output
 
 
-def opd_expand_segments(opd, aperture=None, nterms=15, basis=None,
-                              iterations=2, verbose=False, ignore_border=None, **kwargs):
+def decompose_opd_segments(opd, aperture=None, nterms=15, basis=None,
+                           iterations=2, verbose=False, ignore_border=None, **kwargs):
     """
     Expand OPD into a basis defined by segments, typically with piston, tip, & tilt of each.
```

Comment:
On line 1178, it says `opd_expand_nonorthonormal` when it should now say `decompose_opd_nonorthonormal_basis`